pkg/bot/mattermost.go

```diff
@@ -135,10 +139,25 @@ func NewMattermost(ctx context.Context, log logrus.FieldLogger, commGroupName st
 		botMentionRegex: botMentionRegex,
 		renderer:        NewMattermostRenderer(),
 		userNamesForID:  map[string]string{},
+		messages:        make(chan mattermostMessage, 100),
+		messageWorkers:  pool.New().WithMaxGoroutines(10),
```

Comment:
you have already const in `dicord.go` please make them "global/generic" and use in all places 👍

Reply:
oh, I already used dedicated const for each platform, let me create for mattermost too. So, I assume they might have different configs